dotnet/src/SemanticKernel.Test/TemplateEngine/PromptTemplateEngineTests.cs

```diff
@@ -20,16 +21,18 @@ public sealed class TemplateEngineTests : IDisposable
     private readonly IPromptTemplateEngine _target;
     private readonly ContextVariables _variables;
     private readonly Mock<IReadOnlySkillCollection> _skills;
-    private readonly RedirectOutput _testOutputHelper;
+    private readonly RedirectOutput _redirectOutput;
+    private readonly ILogger _logger;
 
     public TemplateEngineTests(ITestOutputHelper testOutputHelper)
     {
-        this._testOutputHelper = new RedirectOutput(testOutputHelper);
-        Console.SetOut(this._testOutputHelper);
-
-        this._target = new PromptTemplateEngine(ConsoleLogger.Log);
+        this._redirectOutput = new RedirectOutput(testOutputHelper);
+        this._logger = ConsoleLogger.Log;
+        this._target = new PromptTemplateEngine(this._logger);
         this._variables = new ContextVariables(Guid.NewGuid().ToString("X"));
         this._skills = new Mock<IReadOnlySkillCollection>();
+
+        Console.SetOut(this._redirectOutput);
```

Comment:
since the removal of Console.WriteLine from the tests, do we still need this?

Reply:
@dluc Good point, removed `Console.Setout`, and `RedirectOutput` class. They weren't used.